### PR TITLE
LPS-193186 | Test Automation | Master

### PR DESCRIPTION
**Pull request by rafaellajordao**
# LPS-193186 -  Test Automation Creation

Hi! :)
This pr is corresponding to this ticket: [LPS-149468](https://liferay.atlassian.net/browse/LPS-149468) → [LPS-193186](https://liferay.atlassian.net/browse/LPS-193186)

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by rafaellajordao** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationEditSegment.testcase [hunk 1]
```diff
@@ -102,6 +102,24 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-149468. The kebab menu next to a segment must have icons next to the menu items."
+	@priority = 3
+	test CheckIconsInMenuItems {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationEditSegment.testcase:107]: ViewIconsOfSegmentsActions

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationEditSegment.testcase [hunk 2]
```diff
@@ -102,6 +102,24 @@ definition {
...
+		task ("Given the kebab menu next to a segment") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationEditSegment.testcase:108]: When the user opens the dropdown menu of a segment

**Review by Tim-Cao — CHANGES_REQUESTED** [review]
Hi @rafaellajordao, just small changes, Could you made them?

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by Tim-Cao** [comment]
LGTM!

**Comment by Tim-Cao** [comment]
ci:forward

**Pull request closed without merging**